lib/load_aws.rb

```diff
@@ -1,15 +1,18 @@
 require 'yaml'
+require_relative '../config/q_it'
+
+AWS_PATH = File.join(QIt.root, 'config/aws.yml')
 
 def generate_aws_yml_file
   sample_aws = {"aws"=>{"access_key_id"=>"", "secret_access_key"=>"", "region"=>""}}
-  File.open(File.join(Dir.pwd, 'aws.yml'), 'w') do |file|
+  File.open(AWS_PATH, 'w') do |file|
     file.puts sample_aws.to_yaml
   end
 end
 
 begin
-  if File.exists?(File.join(Dir.pwd, 'aws.yml'))
-    AWS = YAML.load(File.read('aws.yml'))["aws"]
+  if File.exists?(File.join(Dir.pwd, 'config/aws.yml'))
```

Comment:
Use `AWS_PATH` here.

Reply:
Have already done that.. let me check in case I forgot to add and push that file..